go.mod

```diff
@@ -6,15 +6,15 @@ require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/hashicorp/go-version v1.6.0
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.29.0
-	github.com/rancher/rke v1.4.10
+	github.com/rancher/rke v1.4.10-rc1
```

Comment:
This does not look right, the robot is not smart enough 😄

Reply:
Thank you for actually reviewing this lol I foolishly trusted automation. Close this out?